src/components/Person.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+const Person = ({
+                  name,
+                  age,
+                  sex,
+                  isMarried,
+                  partnerName
+}) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
```

Comment:
Краще записуй ось так, також виправ це далі в коді)

```suggestion
        {`My name is ${name}`}
```